discount/src/main/java/com/kyriba/conference/discount/api/dto/DiscountResponse.java

```diff
@@ -0,0 +1,23 @@
+package com.kyriba.conference.discount.api.dto;
+
+import com.kyriba.conference.discount.domain.DiscountType;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotNull;
+
+/**
+ * @author Igor Lizura
+ */
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@ApiModel(value = "DiscountType", description = "Discount response")
+public class DiscountResponse {
```

Comment:
This DTO could be embedded into controller and all annotations could be replaced with Value annotation.

Reply:
1. I prefer to keep DTO's in one place
2. Immutability doesn't work with MapStruct

Reply:
Immutability is more important than concrete technology. :)